### DISCUSSION TITLE
替换列头排序svg图标问题, 代码不执行？帮忙看一下呗

### DISCUSSION THREAD

**Discussion by dengyushuai** [antvis/S2#1643] [Q&A] [answered]
这行代码为什么不执行呢？
**codesandbox地址：https://codesandbox.io/s/antv-tou-bu-pai-xu-zxz5jf?file=/src/index.js**

![image](https://user-images.githubusercontent.com/26482193/182552054-ae63b509-8408-4ea6-93b0-083b9e2dcf9f.png)

**Comment by lijinke666** [comment, marked as answer]
https://github.com/antvis/S2/issues/1670
